frontend/src/pages/settings/Settings.action.ts

```diff
@@ -39,5 +42,20 @@ async function patchArchiveConfigAction({ request }: ActionFunctionArgs) {
     }
     throw e;
   }
-  return null;
+  return {
```

Comment:
Even though technically, we can return [anything](https://reactrouter.com/6.30.1/route/action#returning-responses) from an action at use it in `useActionData()`, our `useSubmitAction()` hook wil use this to obtain errors and consider the returned value as such. I think it's okay if our action does not return a value on success for now.